salt/cloud/clouds/vmware.py

```diff
@@ -3023,7 +3023,7 @@ def create_snapshot(name, kwargs=None, call=None):
     .. code-block:: bash
 
         salt-cloud -a create_snapshot vmname snapshot_name="mySnapshot"
-        salt-cloud -a create_snapshot vmname snapshot_name="mySnapshot" [description="My snapshot"] [memdump=False] [quiesce=True]
```

Comment:
``[]`` was specified indicating that these parameters are optional and not required. This optional regex example exists in many Salt and salt-cloud module CLI examples.